plugin crashes when using <includeCommitsAfter>

We run the latest version of the plugin 1.13.15 but it keeps crashing on new GIT repositories for us. Here's a part of what I think is the most important part of the stack of the crash:
```
Caused by: java.lang.ArrayIndexOutOfBoundsException: 0
    at org.eclipse.jgit.revwalk.RevCommit.getParent (RevCommit.java:295)
    at com.github.danielflower.mavenplugins.gitlog.filters.PathCommitFilter.isFoundInPath (PathCommitFilter.java:48)
    at com.github.danielflower.mavenplugins.gitlog.filters.PathCommitFilter.renderCommit (PathCommitFilter.java:40)
    at com.github.danielflower.mavenplugins.gitlog.Generator.show (Generator.java:108)
    at com.github.danielflower.mavenplugins.gitlog.Generator.generate (Generator.java:82)
    at com.github.danielflower.mavenplugins.gitlog.GenerateMojo.execute (GenerateMojo.java:348)
```

You can see that it actually crashes in the jgit plugin. I've been looking into the POM and it seems the plugin is using an older version of the jgit plugin... Not even the original Eclipse one. Should this be bumped to a newer version?

--- a/src/main/java/com/github/danielflower/mavenplugins/gitlog/filters/PathCommitFilter.java
+++ b/src/main/java/com/github/danielflower/mavenplugins/gitlog/filters/PathCommitFilter.java
@@ -45,10 +45,12 @@ public class PathCommitFilter implements CommitFilter {
     }
 
     private boolean isFoundInPath(RevCommit commit) throws IOException {
-        for (DiffEntry diff : getDiffs(commit, commit.getParent(0))) {
-            if (diff.getNewPath().startsWith(relativizedPath)) {
-                return true;
-            }
+        if (commit.getParentCount() > 0) {
+		  for (DiffEntry diff : getDiffs(commit, commit.getParent(0))) {
+              if (diff.getNewPath().startsWith(relativizedPath)) {
+                  return true;
+              }
+		  }
         }
 
         return false;

